packages/aws-cdk/lib/version.ts

```diff
@@ -1,17 +1,21 @@
 import { exec as _exec } from 'child_process';
 import colors = require('colors/safe');
-import { close as _close, open as _open, stat as _stat } from 'fs';
+import fs = require('fs');
```

Comment:
Wait, we have `fs-extra` on purpose so that we don't have to use anything from `fs`.

We generally do something like `import fs = require('fs-extra');`

Reply:
Fixed it up. I didn't realize all `fs` methods are automatically attached to `fs-extra`